admin/lint_json.m

```diff
@@ -0,0 +1,52 @@
+function lint_json(filesin, outputfile)
+    if isempty(filesin)
+        % Default to glob all
+        filesin = "**/*.m";
+    end
+    if isempty(outputfile) % Default to stdout
```

Comment:
If `outputfile` is not specified then this errors. `if exists('outputfile', 'var') && isempty(outputfile)` would default to stdout as described

Reply:
`outputfile` can't be empty though, right? It's a required input arg?

Reply:
I got the logic completely wrong here, sorry. `if ~exists('outputfile', 'var') || isempty(outputfile)` is more like what you'd want.

Reply:
Also `exist` not `exists`... I should really try running this stuff before commenting 🙈

Reply:
All arguments in a Matlab function are "optional". You'll only hit a "Not enough input arguments." error on the first use of an unspecified argument. If the intention is for both arguments to be required, but you specify empty strings for a default, that's OK, but needs to be in the function help.